editor/filesystem_dock.cpp

```diff
@@ -182,16 +182,38 @@ bool FileSystemDock::_create_tree(TreeItem *p_parent, EditorFileSystemDirectory
 	// Create a tree item for the subdirectory.
 	TreeItem *subdirectory_item = tree->create_item(p_parent);
 	String dname = p_dir->get_name();
+	String lpath = p_dir->get_path();
+
 	if (dname.is_empty()) {
 		dname = "res://";
 	}
 
+	// Set custom folder color (if applicable)
```

Comment:
```suggestion
	// Set custom folder color (if applicable).
```
[Comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide) nitpick

Reply:
Fixed in 24487ac1eb9c77a4edbd976bbcdd31d9d082b1c3. Thanks!